tests/parsers/test_orca/test_orca_default.csv

```diff
@@ -0,0 +1,10 @@
+,structure_cell
```

Comment:
I am confused as to where these numbers come from, but I don't think we need this file?

Reply:
Thank you for the explanation. Since we (`aiida-orca` devs) are new into these pytest fixtures / plugins, it would be very useful to include this info somewhere, either as a comment in the conftest file, or in a dev part of the main README perhaps?

Reply:
Will add it to the `README.md` for now. If it becomes too big at some point you can always move it out somewhere else

Reply:
Added